backend/src/model/query/milestone.js

```diff
@@ -0,0 +1,14 @@
+const getMilestone =
+  'SELECT id, title, dueDate, description, isOpen from Milestone'
+const getMilestoneDetail = `select m.id, m.title, m.dueDate, m.description, m.isOpen, (
+  select count(i.id) from Issue as i
+  where m.id = i.milestoneId and i.isOpen = 1
+  ) as openIssue, (
+  select count(i.id) from Issue as i
```

Comment:
`count(*)` 로 했을 때와 무슨 차이가 있는지 궁금합니다!

Reply:
in mysql COUNT(*) with InnoDB is slower than COUNT(ID) because InnoDB doesn't cache the row count.
라는 자료가 있습니다!
단, 이는 mysql의 innoDB 인 경우이고 상황에 따라 다른 것 같습니다.

Reply:
알려주셔서 감사합니다 :D 저희 DB도, mysqld의 default engine도 innoDB라 유용할 것 같네요 👍🏻